src/functions/module_statistical.ts

```diff
@@ -981,20 +984,20 @@ export const SMALL = {
   computeFormat: (data: Arg) => {
     return Array.isArray(data) ? data[0][0].format : data?.format;
   },
-  compute: function (data: ArgValue, n: PrimitiveArgValue): number {
-    const _n = Math.trunc(toNumber(n, this.locale));
-    let largests: number[] = [];
+  computeValueAndFormat: function (data: Arg, n: PrimitiveArg): PrimitiveFunctionReturn {
```

Comment:
computeFormat above can be removed (see my previous comment to add this kind of validation; this proves it would be useful :smile: )